app/helpers/dashboards_helper.rb

```diff
@@ -1,8 +1,8 @@
 # Convenience methods consumed in the dashboards controller index view
 module DashboardsHelper
-  def week_range(date: Time.zone.now, start_day: :monday)
-    week_start = date.beginning_of_week start_day
-    week_end = date.end_of_week start_day
+  def week_range(date: Time.zone.now)
```

Comment:
The default 2nd parameter/default argument wasn't causing a problem, it just became unnecessary since Config.start_day defaults to Monday on it's own.

If I were to worry about optimizing this I'd probably go for memoizing the Config.start_day itself, but more than happy to assign to a local variable since that's easy too.

Reply:
Fixed it to only call once.